docs/guide/plugins/android.md

```diff
@@ -9,9 +9,9 @@ contributors:
 
 # Capacitor Android Plugin Guide
 
-<p class="intro">Building Capacitor plugins for Android involves writing Java to interface with Android SDKs.</p>
+<p class="intro">Building Capacitor plugins for Android involves writing Java or Kotlin to interface with Android SDKs.</p>
 
-<p class="intro">Capacitor embraces standard Android development tools for building Android plugins. We believe that using Java directly will make it easier to use existing solutions on Stack Overflow, share work with existing native developers, and use platform features as soon as they are made available.</p>
+<p class="intro">Capacitor embraces standard Android development tools for building Android plugins. We believe that using Java or Kotlin directly will make it easier to use existing solutions on Stack Overflow, share work with existing native developers, and use platform features as soon as they are made available.</p>
```

Comment:
Let's just delete this and add a section about Java vs Kotlin later.

```suggestion
```

Reply:
This paragraph was there before my edit to add Kotlin. Just wanted to make sure it was intended to remove the whole thing before signing off.